### ISSUE TITLE
MOSEI baseline?

### ISSUE THREAD

**Issue by rhoposit** [A2Zadeh/CMU-MultimodalDataSDK#19]
apologies if this has already been asked.. is there a baseline system that we are compared against, or that is used as a performance cut-off? Also, do you happen to have a standardized "random guess" baseline or approach that everyone should follow, or shall we create our own random guess?

**Comment by A2Zadeh** [comment]
No. Merely compared to other submissions. In general our goal is to diversify the publications as much as possible so even if your method is not performing well. We accept negative results as long as it is conducted correctly.

**Comment by codeislife99** [comment]
The mean intensity value for training set is 0.17 and validation set is 0.15. So I guess in this case of regression, a random baseline would be that the network always predicts the intensity value of 0.16. This could be compared with your ML models.

**Comment by rhoposit** [comment]
@A2Zadeh ok we will work on this baseline and see we can fit a description into the paper

**Comment by codeislife99** [comment]
The random baselines are calculated using training ONE parameter which only sees the outputs. I think this is a solid assumption. So as per my calculations, the random baseline for MSE(sum across 6 categories) is 0.63 on the validation set and 0.64 on the test set. Using the same metric and only the vision modality you would easily get 0.5 MSE(again sum across 6 categories). The random baseline for MAE(sum across 6 categories) is 0.79(Val) and 0.81(test). Hope this helps.

**Comment by rhoposit** [comment]
@codeislife99 cool thanks I'll keep your error values in mind when we develop our random baseline
